app/model.rb

```diff
@@ -17,15 +18,19 @@ class User
 DataMapper.finalize.auto_upgrade!
 
 class Ocr
-  def init
-    @engine = Tesseract::Engine.new {|engine|
-                engine.language = :lol # arbitrary
+  def initialize
+    @engine = ::Tesseract::Engine.new {|engine|
+                engine.language = :eng
                 engine.page_segmentation_mode = 4
-                engine.whitelist = [*'a'..'z', *'A'..'Z', *0..9].join # thanks ruby, made this easy
+                engine.whitelist = [*'a'..'z', *'A'..'Z', *0..9].flatten # thanks ruby, made this easy
```

Comment:
Oh, actually this may work both ways, or I got some tests confused and it was broken for other reasons with the .join, just had it work both ways, but I don't know if you commonly want to whitelist or blacklist like this... but your page segmentation mode may be a problem here, or at any rate commenting it out helped me...  reference here https://tesseract-ocr.googlecode.com/git/doc/tesseract.1.html but still/again playing